editor/editor_build_profile.cpp

```diff
@@ -345,7 +345,7 @@ void EditorBuildProfile::_bind_methods() {
 	BIND_ENUM_CONSTANT(BUILD_OPTION_TEXT_SERVER_ADVANCED);
 	BIND_ENUM_CONSTANT(BUILD_OPTION_DYNAMIC_FONTS);
 	BIND_ENUM_CONSTANT(BUILD_OPTION_WOFF2_FONTS);
-	BIND_ENUM_CONSTANT(BUILD_OPTION_GRPAHITE_FONTS);
+	BIND_ENUM_CONSTANT(BUILD_OPTION_GRAPHITE_FONTS);
```

Comment:
I don't believe this enum is actually exposed, despite being bound (since editor build profile is not exposed), so I do not think this is a breaking change.

Let me know if I missed something, e.g. regarding GDExtension compatibility.

Reply:
Let's see what the docgen says, and the extension

Reply:
Checks seem to pass successfully, so it should be fine?